yarn-project/simulator/src/avm/avm_simulator.ts

```diff
@@ -49,24 +43,35 @@ export class AvmSimulator {
   private tallyPrintFunction = () => {};
   private tallyInstructionFunction = (_a: number, _b: string, _c: Gas) => {};
 
+  // Test Purposes only: Logger will not have the proper function name. Use this constructor for testing purposes
+  // only. Otherwise, use build() below.
   constructor(private context: AvmContext, private instructionSet: InstructionSet = INSTRUCTION_SET()) {
     assert(
       context.machineState.gasLeft.l2Gas <= MAX_L2_GAS_PER_ENQUEUED_CALL,
       `Cannot allocate more than ${MAX_L2_GAS_PER_ENQUEUED_CALL} to the AVM for execution of an enqueued call`,
     );
-    this.log = createLogger(`simulator:avm:core(f:${context.environment.functionSelector.toString()})`);
+    this.log = createLogger(`aztec:avm_simulator:core(calldata[0]: ${context.environment.calldata[0]})`);
```

Comment:
I think Palla just removed all the `aztec:` and used the package name (`simulator`) pls check the recently merged prs